pkg/api/v1/clients.go

```diff
@@ -90,6 +91,37 @@ func (c *ClientRoutes) registerClient(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// unregisterClient
+//
+//	@Summary		Unregister a client
+//	@Description	Unregister a client from ToolHive
+//	@Tags			clients
+//	@Accept			json
+//	@Param			client	body	deleteClientRequest	true	"Client to unregister"
+//	@Success		204
+//	@Failure		400	{string}	string	"Invalid request"
+//	@Router			/api/v1beta/clients [delete]
```

Comment:
We should take the name as a URL parameter, e.g. `DELETE /api/v1beta/clients/{client name}`

Reply:
Good point 👍 I was following what was already done for the register command, do you want me to update both?

Reply:
Typical REST practice is that a POST to create does not take an ID/Name. If we wanted to take a name in the URL, we would use a PUT instead.

tl;dr - leave the POST request as-is.